llvm/lib/Transforms/Utils/ModuleUtils.cpp

```diff
@@ -397,6 +399,98 @@ void llvm::embedBufferInModule(Module &M, MemoryBufferRef Buf,
   appendToCompilerUsed(M, GV);
 }
 
+void llvm::lowerIFuncsOnAIX(Module &M) {
```

Comment:
Should not introduce new target-specific intrinsic uses in lib/Transform. I think the preferred way is to avoid using LowerIFuncPass and adding a new Target/PowerPC pass instead.